### ISSUE TITLE
adjustment of thread density

### ISSUE THREAD

**Issue by ellenha** [Kairotic/patternmatrix2#4]
weft density is currently higher than warp density in the simulation

**Comment by ellenha** [comment]
is it possible to include variation of thread density? For simulating weft-faced patterns for example?

**Comment by nebogeo** [comment]
Yes, indeed - could be done by adding thread type (material/spin and colour) to the colour control block.
